main.rb

```diff
@@ -13,17 +14,6 @@ def options_menu
   puts "\n"
 end
 
-def handle_menu_choice(choice, app)
-  case choice
-  when 1 then app.list_all_books
-  when 2 then app.list_all_people
-  when 3 then app.create_person
-  when 4 then app.create_a_book
-  when 5 then app.create_a_rental
-  when 6 then app.rental_person_id
-  end
-end
-
 def main
   app = App.new
 
```

Comment:
- You have done great so far but there are a few changes you need to make before moving to the next milestone. :smile: :+1: 


- [ ] Follwoing the [SOLID](`https://github.com/microverseinc/curriculum-ruby/blob/main/oop/lessons/solid.md#best-practices-and-solid`) PRINCIPLE AND DRY. You need to refactore your `create_person` and also the `create_student and create_teacher` method. All requests for input and input validation should've been placed in the main.rb file and/or classes/methods whose responsibilities are to request or validate input.
-  Also the input for `Age, name` is repeated in create person and teacher which doesn't follow the dry principle. I'll suggest you take note of these and update your code. :smile: :wave:

Reply:
@V-Blaze thanks for your feedback! We will review these required changes!